cmd/create/cluster/cmd.go

```diff
@@ -1166,26 +1173,10 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
-	var hostedCPPolicies bool
-	if isHostedCP {
```

Comment:
I think we still need the check here, if the `roleARN` was provided in interactive mode, and wasn't available before.